x-pack/plugins/event_log/server/event_log_start_service.ts

```diff
@@ -11,37 +11,45 @@ import {
   SavedObjectsServiceStart,
   SavedObjectsClientContract,
 } from 'src/core/server';
+import { SpacesServiceSetup } from '../../spaces/server';
 
 import { EsContext } from './es';
 import { IEventLogClientService } from './types';
 import { EventLogClient } from './event_log_client';
 export type PluginClusterClient = Pick<LegacyClusterClient, 'callAsInternalUser' | 'asScoped'>;
 export type AdminClusterClient$ = Observable<PluginClusterClient>;
 
+const includedHiddenTypes = ['action', 'alert'];
+
 interface EventLogServiceCtorParams {
   esContext: EsContext;
   savedObjectsService: SavedObjectsServiceStart;
+  spacesService?: SpacesServiceSetup;
 }
 
 // note that clusterClient may be null, indicating we can't write to ES
 export class EventLogClientService implements IEventLogClientService {
   private esContext: EsContext;
   private savedObjectsService: SavedObjectsServiceStart;
+  private spacesService?: SpacesServiceSetup;
 
-  constructor({ esContext, savedObjectsService }: EventLogServiceCtorParams) {
+  constructor({ esContext, savedObjectsService, spacesService }: EventLogServiceCtorParams) {
     this.esContext = esContext;
     this.savedObjectsService = savedObjectsService;
+    this.spacesService = spacesService;
   }
 
-  getClient(
-    request: KibanaRequest,
-    savedObjectsClient: SavedObjectsClientContract = this.savedObjectsService.getScopedClient(
-      request
-    )
-  ) {
+  getClient(request: KibanaRequest) {
+    const savedObjectsClient: SavedObjectsClientContract = this.savedObjectsService.getScopedClient(
```

Comment:
Yes it will break by skipping the privilege checks.  I've updated the title and a little of the description of https://github.com/elastic/kibana/issues/63961, which is where we'll fix this.

I tried to squeeze it in here, but it was getting pretty hairy so I backed out it out.  I'll make some notes in that issue on what I tried.